README.md

```diff
@@ -418,6 +348,10 @@ Here is a list of all the supported methods with their current status:
 | ✅     | `starknet_syncing`                         |
 | ✅     | `starknet_getEvents`                       |
 | ✅     | `starknet_getNonce`                        |
+| ✅     | `starknet_getCompiledCasm` (v0.8.0)        |
+| 🚧     | `getMessageStatus` (v0.8.0)                |
+| 🚧     | `getStorageProof` (v0.8.0)                 |
+| ❌     | `getTransactionStatus` (v0.8.0)            |
```

Comment:
Get transaction status has been introduced way before in v0.6 iirc

Reply:
and we support it correctly

Reply:
This has been fixed. Also added `starknet_` namespace to methods which were missing it,